File: Salario-com-Bonus/SalarioComBonus.java
import java.util.Scanner;

public class SalarioComBonus{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String nome = in.nextLine();
        double salario = in.nextDouble();
        double vendas = in.nextDouble();
        System.out.printf("TOTAL = R$ %.2f\n", salario + vendas * 0.15);
    }
}

File: Salario-com-Bonus/SalarioComBonus.cs
using System;

public class SalarioComBonus{
    public static void Main(){
        string nome = Console.ReadLine();
        double salario = double.Parse(Console.ReadLine());
        double vendas = double.Parse(Console.ReadLine());
        Console.WriteLine($"TOTAL = R$ {salario + vendas * 0.15:0.00}");
    }
}